src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -217,6 +221,7 @@ protected function describeContainerServices(ContainerBuilder $builder, array $o
         $tableRows = array();
         foreach ($this->sortServiceIds($serviceIds) as $serviceId) {
             $definition = $this->resolveServiceDefinition($builder, $serviceId);
+            $styledServiceId = sprintf('<fg=cyan>%s</fg=cyan>', $serviceId);
```

Comment:
This was an extra thing I snuck in to help prettify the output (see screenshot). I want the service id to "jump out" a bit more.